packages/helm-charts/oracle/values.yaml

```diff
@@ -17,14 +17,18 @@ kubectl:
 
 oracle:
   replicas: 2
-  rpcProviderUrl: https://alfajoresstaging-forno.celo-testnet.org
+  rpcProviderUrls:
```

Comment:
This will be moved to the .env file, correct?

Reply:
They are but indirectly- they're overwritten around here https://github.com/celo-org/celo-monorepo/pull/4043/files#diff-78d16d829b51505638952f432154adcaR156 using info from the .env file (like the environment name)